optuna/_dataframe.py

```diff
@@ -0,0 +1,81 @@
+import collections
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Set
+from typing import Tuple
+
+import optuna
+from optuna._imports import try_import
+from optuna.trial._state import TrialState
+
+
+with try_import() as _imports:
+    # `Study.trials_dataframe` is disabled if pandas is not available.
+    import pandas as pd  # NOQA
```

Comment:
[question]

Is `# NOQA` still needed here? (I'm wondering if we can remove it, as there is no variable assignment here.)

Reply:
I think you're right. Removed it.